_sass/components/_side-nav.scss

```diff
@@ -1,3 +1,8 @@
+.usa-sticky-sidenav {
+  position: sticky;
```

Comment:
This is not supported in IE at all, I'm not sure we should be ignoring the browser entirely.

Reply:
And only partial support in Chrome: https://caniuse.com/#search=sticky

Reply:
Counterpoints:
1. That page says it works for 86% of users
2. The 'partial support' relates to an issue not applicable in this instance
3. IE is dead, long live Edge :-)

Reply:
@jordangov also that page is awesome, thanks for the resource!

Reply:
👍 on the page.

As for IE, that's not up to either of us, but rather the USWDS team. That said, I think this feature could also be solved with `position: static` as well? It would require more styling and be trickier (`sticky` is awesome).

Reply:
Static would require some JS shenanigans as I understand it, though my CSS foo may be weak

Reply:
FWIW, I think it's okay for an optional feature to be missing on a single legacy browser

Reply:
@line47 and @hursey013 to make the call

Reply:
Yeah, that's a good point. If it's documented that this (optional) feature is only supported in certain browsers that would fine for me... but again, USWDS will have to decide that. :)

It may be good to confirm that this CSS doesn't negatively impact the page in IE so that we don't need to do any browser hacks. We can't change the jekyll config per browser. :)

Reply:
I think it's okay to use the sticky option as an enhancement @hursey013 what are your thoughts there?